### DISCUSSION TITLE
Project indexing progress reporting?

### DISCUSSION THREAD

**Discussion by keinsell** [zed-industries/zed#6574] [Feature Requests]
As a user, I would like to receive updates on the progress of the indexing process when opening a large project (consisting of 1.2k files and 33.4k lines) in the editor. Currently, the directories are not immediately ready upon opening, which is understandable. However, I would appreciate knowing the status and progress of the indexing operation.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
